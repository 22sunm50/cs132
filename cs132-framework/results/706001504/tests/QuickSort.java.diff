1c1
< minijava.syntaxtree.Goal@446cdf90
---
> Program type checked successfully
