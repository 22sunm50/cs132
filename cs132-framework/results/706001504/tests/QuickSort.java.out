minijava.syntaxtree.Goal@446cdf90
